lib/model/internal_link.dart

```diff
@@ -0,0 +1,282 @@
+
+import 'package:flutter/cupertino.dart';
```

Comment:
nit: This import is broader than needed. In particular it's a widget library, and this file shouldn't need to know anything about widgets.

Android Studio has a habit of inserting this one when what you actually want can be found in a more base-layer library like `flutter/foundation` or `flutter/widgets`. (Perhaps because "cupertino" comes first alphabetically.)